controllers/controllers/resource/testdata/eksa-cluster-vsphere.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: anywhere.eks.amazonaws.com/v1alpha1
```

Comment:
We shouldnt be adding vpshere files. It looks like this PR needs a rebase on top of main on this repo.